README.md

```diff
@@ -19,11 +16,10 @@ Aptos-core strives towards being the safest and most scalable layer one blockcha
 * [Aptos Labs](https://aptoslabs.com/)
 * [Aptos Developer Network](https://aptos.dev)
 * [Getting Started](https://aptos.dev/guides/getting-started)
-* [Life of a Transaction](https://aptos.dev/guides/basics-life-of-txn)
```

Comment:
Hmm?

Reply:
I think the Life of a Transaction link on the README is kinda random. We already have a link to the docs entrypoint and to Getting Started. What makes the "Life of a Transaction" page so special that it needs to be mentioned here?

Reply:
Yeah, I'm not as opinionated about this, but I do think we shouldn't drop our docs pointer, we should take it as a point of action to 1) update it and 2) automate it.